From 479e712a5f628fb9c4c2dd5a9dbff4855b92f1c1 Mon Sep 17 00:00:00 2001
From: goglepox <goglepox@fcb5ba71-d80d-0410-8237-ba3920747fcc>
Date: Tue, 9 Jun 2009 08:10:39 +0000
Subject: [PATCH] Changed int to double

This make it behave much better and switch direction
when a bond is straight down or up.

Not patched in cdk

git-svn-id: https://bioclipse.svn.sourceforge.net/svnroot/bioclipse/bioclipse2/trunk@11152 fcb5ba71-d80d-0410-8237-ba3920747fcc
---
 .../openscience/cdk/geometry/GeometryTools.java    |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/cdk-externals/trunk/org.openscience.cdk.standard/src/org/openscience/cdk/geometry/GeometryTools.java b/cdk-externals/trunk/org.openscience.cdk.standard/src/org/openscience/cdk/geometry/GeometryTools.java
index 1556277..6166c83 100644
--- a/cdk-externals/trunk/org.openscience.cdk.standard/src/org/openscience/cdk/geometry/GeometryTools.java
+++ b/cdk-externals/trunk/org.openscience.cdk.standard/src/org/openscience/cdk/geometry/GeometryTools.java
@@ -1081,9 +1081,9 @@ public class GeometryTools {
 	 *@return            The bestAlignmentForLabel value
 	 */
 	public static int getBestAlignmentForLabel(IAtomContainer container, IAtom atom) {
-		int overallDiffX = 0;
+		double overallDiffX = 0;
 		for (IAtom connectedAtom : container.getConnectedAtomsList(atom)) {
-			overallDiffX = overallDiffX + (int) (connectedAtom.getPoint2d().x - atom.getPoint2d().x);
+			overallDiffX = overallDiffX +  (connectedAtom.getPoint2d().x - atom.getPoint2d().x);
 		}
 		if (overallDiffX <= 0) {
 			return 1;
-- 
1.6.0.4

